Update example config for Nuxt 4

GigantPro opened the pull request:
- Updated the version with maximum compatibility
- Fixed paths to static files
- Adjusted Nuxt config to ensure proper static site builds

Summary: Update docs for using tauri with Nuxt 4

<!-- Thank you for opening a PR! We really appreciate you taking the time to help out 🙌 -->

#### Description
<!-- Translators, try to follow this pattern when naming your PRs:
"i18n(language code): (short description)" -->
 <!-- If it's an update try adding the commits as reference -->

<!--
Here's what will happen next:

1. Our GitHub bots will run to check your changes.
   If they spot any broken links you will see some error messages on this PR.
   Don't hesitate to ask any questions if you're not sure what these mean!

2. In a few minutes, you'll be able to see a preview of your changes on Netlify 🥳

3. One or more of our maintainers will take a look and may ask you to make changes.
   We try to be responsive, but don't worry if this takes a day or two.

4. Reach out to us on Discord with any questions along the way:
   https://discord.com/invite/tauri
-->

GigantPro committed:
Update example config for Nuxt 4

- Updated the version with maximum compatibility
- Fixed paths to static files
- Adjusted Nuxt config to ensure proper static site builds

FabianLars commented on src/content/docs/start/frontend/nuxt.mdx:
```diff
@@ -64,7 +64,7 @@ Nuxt is a meta framework for Vue. Learn more about Nuxt at https://nuxt.com. Thi
             "beforeDevCommand": "pnpm dev",
             "beforeBuildCommand": "pnpm build",
```
npm yarn and pnpm here use `build` while deno uses `generate`, which one is correct?

GigantPro replied:
Indeed! That’s the core mistake. `nuxt generate` builds the app as a static site, while `nuxt build` builds it as an executable server. Therefore, with yarn/npm/pnpm the command listed is incorrect. If you specify the correct command and also set `ssr: false`, everything will work properly. There’s no need to provide separate Nitro settings for this.

FabianLars commented on src/content/docs/start/frontend/nuxt.mdx:
```diff
@@ -115,6 +115,15 @@ Nuxt is a meta framework for Vue. Learn more about Nuxt at https://nuxt.com. Thi
+          // Set a hard path for statics
+          publicDir: './.output/public'
+        }
+      }
```
can't it inherit if from ssr: false above (which btw is the only thing they documented here https://nuxt.com/docs/4.x/getting-started/deployment#client-side-only-rendering)

GigantPro committed:
Update frontendDist path and build commands in nuxt.mdx

- Fix commands for generater html
- Return correct path (../.output/public -> ../dist)

FabianLars reviewed (approved):
nuxt is such a blackbox for me :D i didn't test these changes but i'm gonna trust you :P Thanks for the PR!
